spotify/__init__.py

```diff
@@ -1,3 +1 @@
-from spotify.app import create_app
-
-APP = create_app()
+from spotify.app import app
```

Comment:
Why does init.py only have the one import statement?
Just feels a bit under utilized to me.

Reply:
__init__.py  Is actually technically worthless in our case. We wrapped the project in a module so the additional py files we import from are easier to manage and we don't get circular imports (ie an import importing something which imports that which imports it) and __init__.py  is a formality with this package structure. It's standard.